### PR TITLE
[MOB-4356] react native inbox

### PR DESCRIPTION
**Pull request by evangreer91**
## 🔹 JIRA Ticket(s) if any

* [MOB-4356] (https://iterable.atlassian.net/browse/MOB-4356)

## ✏️ Description

This pull request includes the release of the react native mobile inbox.

[MOB-4356]: https://iterable.atlassian.net/browse/MOB-4356?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ts/IterableInbox.tsx [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+      container,
+      headline,
+      messageListContainer
+   } = styles
+
+   const navTitleHeight = headline.height + headline.paddingTop + headline.paddingBottom
+   headline = { ...headline, height: Platform.OS === "android" ? 70 : 60 }
+   headline = (!isPortrait) ? { ...headline, paddingLeft: 70 } : headline
+
+   useEffect(() => {
```
Comment (roninopf) [inline ts/IterableInbox.tsx:119]: Let's leave a comment on each `useEffect` for future reference.

File: ts/IterableInbox.tsx [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+      fetchInboxMessages()
+      addInboxChangedListener()
+
+      return () => {
+         removeInboxChangedListener()
+         inboxDataModel.endSession(visibleMessageImpressions)
+         console.log("end session")
```
Comment (roninopf) [inline ts/IterableInbox.tsx:126]: We can remove these log prints now, this was leftover during debugging.

**Review by roninopf — APPROVED** [review]

File: ts/IterableInbox.tsx (hunk 2 above)
Comment (evangreer91) [reply ts/IterableInbox.tsx:126]: I will remove.

File: ts/IterableInbox.tsx (hunk 1 above)
Comment (evangreer91) [reply ts/IterableInbox.tsx:119]: Adding comments.

**Pull request merged**